ross/tests/test_cylindrical.py

```diff
@@ -0,0 +1,153 @@
+import math
+
+import numpy as np
+import pytest
+from numpy.testing import assert_allclose, assert_almost_equal
+
+from ross.fluid_flow.cylindrical import THDCylindrical
+from ross.units import Q_
+
+if __name__ == "__main__":
+
+    x0 = [0.1, -0.1]
+    L = float(0.263144)  # [metros]
+    R = float(0.2)  # [metros]
+    Cr = float(1.945e-4)  # [metros]
+    nTheta = int(41)
+    nZ = int(10)
+    nY = None
+
+    mu = float(0.02)  # [Ns/m²]
+    speed = Q_(900, "RPM")  # [RPM]
+    Wx = float(0)  # [N]
+    Wy = float(-112814.91)  # [N]
+    k = float(0.15327)  # Thermal conductivity [J/s.m.°C]
+    Cp = float(1915.5)  # Specific heat [J/kg°C]
+    rho = float(854.952)  # Specific mass [kg/m³]
+    Treserv = float(50)  # Temperature of oil tank [ºC]
+    mix = float(0.52)  # Mixing factor. Used because the oil supply flow is not known.
+    nGap = int(1)  #    Number of volumes in recess zone
+    nPad = int(2)  #    Number of pads
+    betha_s = 176
+    T_1 = float(50)
+    T_2 = float(80)
+    mu_ref1 = float(0.02)
+    mu_ref2 = float(0.01)
+
+    mancal = THDCylindrical(
+        L,
+        R,
+        Cr,
+        nTheta,
+        nZ,
+        nY,
+        nGap,
+        betha_s,
+        mu,
+        speed,
+        Wx,
+        Wy,
+        k,
+        Cp,
+        rho,
+        Treserv,
+        mix,
+        T_1,
+        T_2,
+        mu_ref1,
+        mu_ref2,
+    )
+    mancal.run(x0, print_progress=True)
```

Comment:
@elyqg , I think you could remove this part, since it is already present at each pytest fixture.

Reply:
We'll do it right away!